app/test_main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+from unittest import mock
+from app.main import can_access_google_page
```

Comment:
add blank line between this 2 imports

Reply:
still not fixed 

![image](https://user-images.githubusercontent.com/104218550/221216360-d8aaa6f4-7d4d-47e3-b389-18941b79aa37.png)